### ISSUE TITLE
Breadcrumb question

### ISSUE THREAD

**Issue by vivkay** [vivkay/portfolio-new#2]
Hi @thomasjbradley,

I want to add breadcrumbs on each portfolio piece. Of course, I could hand-code these, but I am wondering if there is a way to do this with jekyll (and thus leave less maintenance for future uploads).

Here is a screenshot of what I want: 
![screen shot 2015-10-31 at 1 18 07 pm](https://cloud.githubusercontent.com/assets/6340292/10864889/2d877538-7fd2-11e5-92d4-a651b2ae7ef6.png)

It's also something that Google Material Design has on their pages. 

Basically I want the back button to go to the home page. The forward button I would like to go to the next portfolio page. They are separate html files (each page is unique) but each has a corresponding md file. 

Now I'm starting to confuse myself! Haha, maybe it's not possible. I basically want to tell it "pick one of these five .html files, and cycle through it on the click of this button".

I hope that made sense!

HAPPY HALLOWEEN!

**Comment by thomasjbradley** [comment]
So the red button will always goes back to the portfolio index page? Then, it's URL just needs to be:

```html
{{site.baseurl}}/#portfolio
```

And for the next project, since you're using posts to generate the pages, you should be able to use `page.next`, like this:

```html
{{site.baseurl}}{{page.next}}
```

And since both of those buttons are included in "case study a" and "case study" they should be put into an include so the same code isn't in two places.

**Comment by vivkay** [comment]
Thanks so much Thomas, I will try this soon!
